### DISCUSSION TITLE
Disable Warning  caused in creaction of a saved_object. The warning appears when I transform data by a filtering

### DISCUSSION THREAD

**Discussion by LombardoAndrea195** [vega/altair#2490] [Q&A]
I decided to use the suggestion on this [link](https://www.elastic.co/blog/how-to-jupyter-notebook-visualizations-kibana-dashboards-vega-data-science) that suggest to create panel of vega with altair. Everything works fine except for the warning. I've tried to remove warnings on setting advance but I dindn't mange to solve. 
In order to save the chart I use the routines on this [link](https://github.com/walterra/jupyter2kibana/blob/master/kibana_vega_util.py).
 Every help will be appreciate.
  The problem as suggest in previous topics open is on data, (because the warning is refer to @timestamp and another date field).
   I would like to hide the warning in order to work without it, maybe is it possible inserting a script on the routines on 2 [links](https://discuss.elastic.co/t/infinite-extent-for-field-count-infinity-infinity/276333/3) in order to add the script in this link

**Comment by jakevdp** [comment]
It's unclear what warning you're talking about. Can you provide an example of the code you're running, and the output you're seeing?

**Comment by LombardoAndrea195** [comment]
I don't know if you have also a suggestion for displaying in the legend the colors that appear in the graph.

**Reply by jakevdp** [reply]
I think the issue is that the dataset is initially empty, and then Vega-Lite issues a warning in Javascript. I don't think there's anythiing we can do about that from the Python side, aside from changing your filters such that the default dataset isn't empty.

**Reply by LombardoAndrea195** [reply]
There is a way to disable it?

**Reply by jakevdp** [reply]
Not that I know of

**Reply by LombardoAndrea195** [reply]
Is it possible to modify the script for generating the saved_object in vega for .kibana in order to inser hideWarning=True as suggested in a link above?

**Reply by jakevdp** [reply]
That may be possible. I've never used kibana and am unfamiliar with it. You might get more mileage asking in kibana forums.
